src/ontology/mondo-edit.obo

```diff
@@ -74363,18 +74355,25 @@ subset: rare
 synonym: "malignant ovarian mixed epithelial neoplasm" EXACT [NCIT:C40090]
 synonym: "malignant ovarian mixed epithelial tumor" EXACT [NCIT:C40090]
 synonym: "malignant ovarian mixed epithelial tumour" EXACT OMO:0003005 []
-synonym: "mixed epithelial carcinoma of ovary" EXACT [NCIT:C40090]
+synonym: "mixed epithelial carcinoma of ovary" EXACT [DOID:6898]
 synonym: "mixed epithelial carcinoma of the ovary" EXACT [NCIT:C40090]
+synonym: "mixed ovarian carcinoma" RELATED [ONCOTREE:MXOV]
 synonym: "ovarian mixed epithelial carcinoma" EXACT [NCIT:C40090]
 synonym: "ovarian Seromucinous carcinoma" EXACT [NCIT:C40090]
+synonym: "ovary mixed epithelial carcinoma" EXACT [DOID:6898, MONDO:0004031]
+xref: DOID:6898 {source="MONDO:equivalentTo"}
 xref: DOID:6899 {source="MONDO:equivalentObsolete"}
 xref: MEDGEN:1651770 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: NCIT:C40090 {source="DOID:6899", source="MONDO:equivalentTo"}
+xref: NCIT:C40090 {source="ONCOTREE:MXOV"}
```

Comment:
duplicated with above. I think a normalization step will do the trick

Reply:
sure, are you leaving that for me to update?